src/types.ts

```diff
@@ -27,9 +27,9 @@ export interface MeteoalarmCardConfig extends LovelaceCardConfig {
 
 export interface MeteoalarmIntegration {
   metadata: MeteoalarmIntegrationMetadata,
-  supports(entity: HassEntity): boolean,
+  supports(hass: HomeAssistant, entity: HassEntity): Promise<boolean>,
   alertActive(entity: HassEntity): boolean,
```

Comment:
I like that you have moved all integration methods to be async but you can also do that `alertActive` It doesn't make sense that every method except this one is asynchronous